src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import cn from 'classnames';
+import { SortType } from '../../types/SortType';
+
+interface Props {
+  activeTodosAmount: number;
+  onSort: (sortType: SortType) => void;
+  sortType: SortType;
+  onClearCompletedTodos: () => void;
+  isCompletedTodo: boolean;
+}
+export const Footer: React.FC<Props> = ({
+  activeTodosAmount,
+  onSort,
+  sortType,
+  onClearCompletedTodos,
+  isCompletedTodo,
+}) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {`${activeTodosAmount} items left`}
+    </span>
+
+    <nav className="filter">
+      <a
+        href="#/"
+        className={cn('filter__link', {
+          selected: sortType === SortType.ALL,
+        })}
+        onClick={() => onSort(SortType.ALL)}
+      >
+        All
+      </a>
```

Comment:
As a suggestion, we can map here `SortType` to get rid of duplicate code.
```suggestion
   {Object.values(SortType).map(filterType=> (
      <a
          key={filterType}
          href="#/"
          className={cn('filter__link', {
            selected: filterBy === filterType,
          })}
          onClick={() => setFilterBy(filterType)}
        >
          {filterType}
        </a>
      ))}
```

Reply:
Thank you a lot! These suggestions are very useful!👍